mathics/builtin/list/rearrange.py

```diff
@@ -1388,12 +1402,18 @@ class Union(_SetOperation):
           will be sorted and each element will only occur once.
     </dl>
 
-    >> Union[{5, 1, 3, 7, 1, 8, 3}]
-     = {1, 3, 5, 7, 8}
-
+    A union of two lists:
     >> Union[{a, b, c}, {c, d, e}]
      = {a, b, c, d, e}
 
+    A union of two associations:
+    >> Union[{a -> b}, {c -> d}]
+     = {a -> b, c -> d}
+
+    A union of one item is the item, however note that the list is sorted:
```

Comment:
```suggestion
    A union of one item is the item, however note that the list is sorted and reduced to have unique elements:
```

Reply:
There is something that feels weird to me about combining the two ideas in one sentence like this. In a union, it is expected that duplicates are removed. However, it is not expected that the result will be sorted by value. 

I have reworked the existing examples so that there is one example to show just the sorting without duplication removal.  And then a later example can just mention that duplicates are removed in sets one expects to happen in a union operation.